pkg/security/resolvers/process/resolver_windows.go

```diff
@@ -245,3 +268,31 @@ func (p *Resolver) Snapshot() {
 		p.insertEntry(e)
 	}
 }
+
+// GetUser returns the username
+func (p *Resolver) GetUser(ownerSidString string) (name string) {
```

Comment:
I would create a dedicated resolver like we have a Linux. So basically a Windows file in this package https://github.com/DataDog/datadog-agent/blob/main/pkg/security/resolvers/usergroup/resolver.go. I would do that as there is nothing process specific in this function and it could be used by FIM events later on. Otherwise LGTM